Client/Frontend/Settings/TranslationSettingsController.swift

```diff
@@ -98,17 +80,23 @@ class TranslationSettingsController: ThemedTableViewController {
             if indexPath.row == 0 {
                 cell.textLabel?.text = Strings.SettingTranslateSnackBarSwitchTitle
                 cell.detailTextLabel?.text = Strings.SettingTranslateSnackBarSwitchSubtitle
-                cell.detailTextLabel?.numberOfLines = 2
+                cell.detailTextLabel?.numberOfLines = 4
                 cell.detailTextLabel?.minimumScaleFactor = 0.5
                 cell.detailTextLabel?.adjustsFontSizeToFitWidth = true
 
                 let control = UISwitchThemed()
-
                 control.accessibilityIdentifier = "TranslateSwitchValue"
                 control.onTintColor = UIColor.theme.tableView.controlTint
                 control.addTarget(self, action: #selector(switchValueChanged), for: .valueChanged)
                 control.isOn = setting.translateOnOff
-                cell.accessoryView = control
+
+                // Add spacing between label and switch by wrapping the switch in a view.
+                cell.accessoryView = UIView(frame: CGRect(width: 64, height: 44))
```

Comment:
There aren't handy UISwitch size constants I could use here, these size values are arbitrarily larger than the switch.